src/main/java/com/google/codeu/data/Event.java

```diff
@@ -18,6 +18,17 @@
    * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
    * random ID and uses the current system time for the creation time.
    */
+  public Event(){
```

Comment:
Is all of this necessary? This doesn't have this in the Message.java file, so could you explain why it is?